### PR TITLE
Feature/arm requirements extended

### PR DESCRIPTION
**Pull request by LarsJanssenTUe**

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
@LarsJanssenTUe conflicts from the start 👍🏻

**Comment by LarsJanssenTUe** [comment]
> @LarsJanssenTUe conflicts from the start 👍🏻

@MatthijsBurgh should be good now!

**Review by MatthijsBurgh — APPROVED** [review]
Wait for second approval before merging

**Review by PetervDooren — DISMISSED** [review]
Couple of comments. Over all I think it looks good.
Conceptually it does not sit right with me that arm requirements are specified for the statemachines rather than the individual arms used by the statemachines but I think that will be a topic for another day.

File: robot_skills/src/robot_skills/arm/handover_detector.py
Comment (MatthijsBurgh) [reply robot_skills/src/robot_skills/arm/handover_detector.py]: What do you think of this one. Small change, but enough change IMO.

File: challenge_cleanup/src/challenge_cleanup/self_cleanup.py
Comment (MatthijsBurgh) [reply challenge_cleanup/src/challenge_cleanup/self_cleanup.py:330]: True. Normally each pick-up and place operation should be done with the same arm, though the same designator instance. Other combo operations, can be done with another arm. Though another designator.

But indeed in this situation one designator should be used.

File: robot_skills/src/robot_skills/robot.py
Comment (MatthijsBurgh) [reply robot_skills/src/robot_skills/robot.py:9]: I think this is a good pratice.

**Review by MatthijsBurgh — DISMISSED** [review]
See https://github.com/tue-robotics/tue_robocup/pull/1092/files/0f5dd247a3b6b46855a49240d2b53bdfc98d12f9#r615438247

**Review by PetervDooren — APPROVED** [review]
Tentative approve. I cant clearly see the changes compared to master

**Pull request merged**
